apps/actors/lib/actors.ex

```diff
@@ -197,9 +197,6 @@ defmodule Actors do
       min_demand: @activate_actors_min_demand,
       max_demand: @activate_actors_max_demand
     )
-    |> Flow.filter(fn {_actor_name, %Actor{persistent: persistent} = _actor} ->
```

Comment:
@sleipnir I had to remove this due to breaking tests

The test:

`invoke function for a new actor without persistence in another node`

Couldn't find the actor in the other node due to this persistent filter, idk if there is a better solution here, but maybe its worth solving in another commit

Reply:
Actually I think the test is wrong and not here.